internal/cmd/auth.go

```diff
@@ -346,15 +349,30 @@ func logout(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("could not retrieve local config: %w", err)
 	}
 
-	token := settings.GetToken()
-	if len(token) == 0 {
+	if token := settings.GetToken(); len(token) == 0 {
 		fmt.Println("No user logged in.")
-	} else {
-		settings.SetToken("")
-		settings.SetUsername("")
-		fmt.Println("Logged out.")
+		return nil
 	}
 
+	if flags.All() {
```

Comment:
nit: extract all the related logic to a function

Reply:
thanks! it is way better like this